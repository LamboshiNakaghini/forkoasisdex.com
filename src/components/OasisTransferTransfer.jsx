import React, { PureComponent } from 'react';
import { PropTypes } from 'prop-types';
// import ImmutablePropTypes from 'react-immutable-proptypes';

import styles from './OasisTransferTransfer.scss';

const propTypes = PropTypes && {};
const defaultProps = {};

class OasisTransferTransfer extends PureComponent {
  render() {
    return (
      <div className={styles.base}>
      </div>
    );
  }
}

OasisTransferTransfer.displayName = 'OasisTransferTransfer';
OasisTransferTransfer.propTypes = propTypes;
OasisTransferTransfer.defaultProps = defaultProps;
export default OasisTransferTransfer;
