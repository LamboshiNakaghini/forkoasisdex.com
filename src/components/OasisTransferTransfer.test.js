/* global shallow describe it expect */
/* eslint-disable import/first */
import React from 'react';
import { shallow } from 'enzyme';
import OasisTransferTransfer from './OasisTransferTransfer';

describe('(Component) OasisTransferTransfer', () => {
  it('should render', () => {
    const props = {
      children: (
        <span>test</span>
      ),
    };
    const wrapper = shallow(
      <OasisTransferTransfer {...props}/>,
    );

    expect(wrapper).toMatchSnapshot();
  });
});
