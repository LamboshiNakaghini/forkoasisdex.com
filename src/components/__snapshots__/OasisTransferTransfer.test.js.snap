// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) OasisTransferTransfer should render 1`] = `<div />`;
